src/constants.js

```diff
@@ -98,7 +98,9 @@ export default {
         INSTANT_LAUNCH_LISTING: "listing",
     },
     VICE_LOGS_POLLING_INTERVAL: 10000,
-    DATA_STORAGE_QUOTA_LIMIT: 2 ** 30 * 100, // 100 GiB
+    DATA_STORAGE_QUOTA_LIMIT_DEFAULT: 50 * 2 ** 30,
```

Comment:
Why the change to 50? I thought the default was still 100, but I probably missed something, heh.

Reply:
The limit associated with the basic plan was changed to 50 GiB.